src/tests/testSetup.ts

```diff
@@ -61,12 +79,21 @@ export const runMutationTest = async (
 	const actualContent = await readFile(actualFileName);
 	const expectFileName = `expected.ts${fileNameSuffix}`;
 	const expectedFilePath = path.join(dirPath, expectFileName);
+	const optionsSnapshot = JSON.stringify(pendingOptionsList, null, 2);
 
-	const optionsSnapshot = JSON.stringify(
-		pendingOptionsList,
-		null,
-		2,
-	).replaceAll(dirPath, "<rootDir>");
+	return {
+		actualContent,
+		expectedFilePath,
+		options: replaceFilePath(optionsSnapshot),
+		output: cliOutput.join("\n"),
+	};
+};
 
-	return { actualContent, expectedFilePath, options: optionsSnapshot };
+export const checkTestResult = async (cwd: string, caseDir: string) => {
+	const fullPath = path.join(cwd, "cases", caseDir);
+	const { actualContent, expectedFilePath, options, output } =
+		await runMutationTest(fullPath);
+	await expect(actualContent).toMatchFileSnapshot(expectedFilePath);
+	expect(options).toMatchSnapshot("options");
+	expect(output).toMatchSnapshot("output");
```

Comment:
[Refactor] Having multiple assertions in a test means we only get insights from one failure at a time. If assertion 1 of 3 fails, we don't know at a glance if 2 or 3 did.

My normal trick for this kind of thing is:

```suggestion
	await expect({ actualContent, options, output }).toMatchFileSnapshot(expectedFilePath);
```